cmd/cancel.go

```diff
@@ -69,7 +75,7 @@ func newCancelCmd() *cobra.Command {
 
 			msg := fmt.Sprintf("%sThe currently running update for '%s' has been canceled!%s", colors.SpecAttention, s.Name(),
 				colors.Reset)
-			fmt.Println(colors.ColorizeText(msg))
```

Comment:
no one should call colors.ColorizeText now. Only the very lowest level (that is calling into 'lorely') now does this.

Reply:
Is it possible to stop exporting it?  A brief look at the code makes me think we may be able to do but I am not sure.